src/main/kotlin/io/titandata/titan/Cli.kt

```diff
@@ -32,7 +34,19 @@ object Cli {
 
     @JvmStatic
     fun main(args: Array<String>) {
+        val titanDebug = System.getenv("TITAN_DEBUG")
+        if (titanDebug != null) {
+            val logging = HttpLoggingInterceptor()
+            if (titanDebug.equals("trace", true)) {
+                logging.level = HttpLoggingInterceptor.Level.BODY
+            } else {
+                logging.level = HttpLoggingInterceptor.Level.BASIC
+            }
+            ApiClient.builder.addInterceptor(logging)
+        }
+
         BasicConfigurator.configure();
```

Comment:
We're kind of stacking logging here. What do you think about using slf4j since it's already required by the AWS SDK?

Reply:
OKHttp does use slf4j. This is just how you add the interceptor to enable it.